packages/react/src/global.js

```diff
@@ -91,7 +91,8 @@ export let Global: React.AbstractComponent<GlobalProps> =
     useInsertionEffect(() => {
       const key = `${cache.key}-global`
 
-      let sheet = new StyleSheet({
+      // use case of https://github.com/emotion-js/emotion/issues/2675
+      let sheet = new cache.sheet.constructor({
```

Comment:
Could you remove this import?
https://github.com/emotion-js/emotion/blob/e638c6d844db8c5f33250a830f37c0cd91fdc1e7/packages/react/src/global.js#L8